core-rust/core-api-server/core-api-schema.yaml

```diff
@@ -5426,6 +5435,7 @@ components:
         status:
           $ref: "#/components/schemas/LtsCommittedTransactionStatus"
         fungible_entity_balance_changes:
+          # TODO(during review): is this description still true? we have improved both fees and balance changes recently...
```

Comment:
The node logic seems right - unless something in RE makes it inaccurate (doubt it). It's most probably a leftover.